examples/flux-source/app/index.ts

```diff
@@ -0,0 +1,51 @@
+import * as k8s from "@pulumi/kubernetes";
+
+
+// A git repository source for our Pulumi program
+const gitRepo = new k8s.apiextensions.CustomResource("pko-dev", {
+    apiVersion: 'source.toolkit.fluxcd.io/v1beta2',
+    kind: 'GitRepository',
+    metadata: {
+        'namespace': 'default',
+    },
+    spec: {
+        interval: '5m0s',
+        url: 'https://github.com/squaremo/pko-dev',
+        ref: { branch: 'main' },
```

Comment:
Worth showing off any of the value add here?  (Features we get via Flux that aren’t available directly?)

Reply:
Yes, somewhere. This example is a "how to", to show people how to get it working at all, and (secondarily) for them to adapt to their needs. Putting fancy stuff in would obscure that.

Reply:
It would cool to do more examples that show off e.g.,

 - using OCI images with cosign verification
 - following a semver tag
 - using "ignore" to filter down the files made available
 - using webhooks for notifications (at least once watching sources is implemented in the operator)